front/app/utils/dateUtils.ts

```diff
@@ -313,9 +313,7 @@ export const userTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
 // This function makes sure that the date is always interpreted as midnight in the user's timezone.
 const backendDatestringRegex = /^\d{4}-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01])$/;
 
-export const parseBackendDateString = (_dateString?: string) => {
```

Comment:
Slightly changed how this function works, because it was annoying me that if `_dateString` was 100% certain a string, the return value was still `Date | undefined`.